nevergrad/functions/functionlib.py

```diff
@@ -169,6 +174,7 @@ def __init__(  # pylint: disable=too-many-arguments
         # record necessary info and prepare transforms
         self._dimension = block_dimension * num_blocks + useless_variables
         self._func = corefuncs.registry[name]
+        self._bounded = bounded
```

Comment:
```suggestion
```
not necessary because the information is already available (= when self._transf is defined)